shared/transforms/xccdf-addremediations.xslt

```diff
@@ -71,7 +73,9 @@
     - NIST-800-53-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_nist800_171">
     - NIST-800-171-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_pci_dss">
     - PCI-DSS-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_cjis">
-    - CJIS-<xsl:value-of select="."/></xsl:for-each></xsl:variable>
+    - CJIS-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_disa_ossrg">
+    - DISA-OS-<xsl:value-of select="."/></xsl:for-each><xsl:for-each select="$ref_disa_stigid">
```

Comment:
``DISA-OS-SRG`` might be better

Reply:
Then the actual generated playbooks would look like:
```
        ...
        - DISA-OS-SRG-SRG-OS-000378-GPOS-0016
        - DISA-OS-SRG-SRG-OS-000480-GPOS-00227
        ...
```

SRG-OS is already part of the reference. It's a prefix to all of them.

Reply:
Removed the OS Prefix as well. Now the generated playbooks look like this:
```
        - NIST-800-53-IA-3
        - NIST-800-171-3.1.21
        - DISA-SRG-OS-000114-GPOS-00059
        - DISA-SRG-OS-000378-GPOS-0016
        - DISA-SRG-OS-000480-GPOS-00227
        - DISA-STIG-RHEL-07-020100
```